torchtune/models/llama3_2_vision/_convert_weights.py

```diff
@@ -148,8 +148,8 @@ def llama3_vision_tune_to_meta(
 
     # Calculate fusion_interval: layer interval where cross attention layers are fused
     num_layers = max(_layer_num(k) for k in state_dict if "layers" in k) + 1
-    num_fusion_layers = (
-        max(_layer_num(k) for k in state_dict if "cross_attention_layers" in k) + 1
+    num_fusion_layers = len(
+        set([k.split(".")[2] for k in state_dict if "fusion_layer" in k])
```

Comment:
can you add a comment explaining what the FQN looks like here? Why not just count number of `"fusion_layer" in k for k in state_dict`?

Reply:
> Why not just count number of "fusion_layer" in k for k in state_dict?

There are multiple params for each layer so we need to dedup